smoketest.sh

```diff
@@ -57,7 +56,12 @@ git -C $UPSTREAM_BARE_REPO init --bare
 ./dstask next
 ./dstask 1 done
 ./dstask show-resolved
+
+# TODO we set FAKE_PTY because we do not have a non-tty
+# rendering scheme for show-projects
+export DSTASK_FAKE_PTY=1
 ./dstask show-projects
+unset DSTASK_FAKE_PTY
```

Comment:
It seems show-projects is the only command that still requires the DSTASK_FAKE_PTY variable. We might open an issue to provide a scriptable output for this command, as we've done with others. JSON is what we've done for the others, and that might work here, as well.

Reply:
https://github.com/naggie/dstask/issues/63